legend-engine-xts-json/legend-engine-xt-json-runtime/src/main/java/org/finos/legend/engine/external/format/json/JsonSchemaRuntimeExtension.java

```diff
@@ -125,6 +128,19 @@ public Result executeExternalizeExecutionNode(ExternalFormatExternalizeExecution
         }
     }
 
+    @Override
+    public Result executeExternalizeTDSExecutionNode(ExternalFormatExternalizeTDSExecutionNode node, Result result, Identity identity, ExecutionState executionState)
+    {
+        if (result instanceof RelationalResult)
```

Comment:
This is not a good approach as it does not respect modularization, exposing relational concepts into the external format modules.  

It also limits the externalization to relational, and does not respect other stores the platform support (mongo, elastic)